pkgs/by-name/li/libpsl/package.nix

```diff
@@ -23,9 +25,10 @@ stdenv.mkDerivation rec {
     hash = "sha256-mp9qjG7bplDPnqVUdc0XLdKEhzFoBOnHMgLZdXLNOi0=";
   };
 
-  outputs = [ "out" "dev" ]
-    # bin/psl-make-dafsa brings a large runtime closure through python3
-    ++ lib.optional (!stdenv.hostPlatform.isStatic) "bin";
```

Comment:
Actually it's not a runtime dependency... Well. Maybe we can keep it for native compilation and only do it for cross.